custom-plugins/grafana-worldmap-panel-msupply/src/worldmap_ctrl.ts

```diff
@@ -1,60 +1,58 @@
-import { MetricsPanelCtrl } from "grafana/app/plugins/sdk";
-import TimeSeries from "grafana/app/core/time_series2";
+import { MetricsPanelCtrl } from 'grafana/app/plugins/sdk';
+import TimeSeries from 'grafana/app/core/time_series2';
 import appEvents from 'grafana/app/core/app_events';
 
-import * as _ from "lodash";
-import DataFormatter from "./data_formatter";
-import "./css/worldmap-panel.css";
-import $ from "jquery";
-import "./css/leaflet.css";
-import WorldMap from "./worldmap";
+import * as _ from 'lodash';
```

Comment:
I notice most of the changes here are just minor re-formatting/style changes. I assume your chosen linter is just doing its thing, but should we be using a synced Prettier file or something to maintain consistency here? What's the best way to handle this going forward?

Reply:
oops. yes. just the defaults in my vscode which are causing the problems. I probably should have disabled. will look at adding in configs.